docs/crd.md

```diff
@@ -34,6 +34,7 @@ FlinkCluster
         |__ volumes
         |__ volumeMounts
         |__ nodeSelector
+        |__ tolerations
```

Comment:
Need to update `taskManager` as well. I just noticed `nodeSelector` is missing from `taskManager`, could you add it?